app/emoji.go

```diff
@@ -227,12 +229,17 @@ func (a *App) SearchEmoji(name string, prefixOnly bool, limit int) ([]*model.Emo
 // GetEmojiStaticUrl returns a relative static URL for system default emojis,
 // and the API route for custom ones. Errors if not found or if custom and deleted.
 func (a *App) GetEmojiStaticUrl(emojiName string) (string, *model.AppError) {
+	subPath, _ := utils.GetSubpathFromConfig(a.Config())
+	if subPath == "" {
```

Comment:
Suggest that instead of assigning `/` back to `subPath`, just change the `path.Join` below as proposed.